### DISCUSSION TITLE
【deadlock】Something worng after update

### DISCUSSION THREAD

**Discussion by zjssun** [SteamDatabase/ValveResourceFormat#880] [General]
![屏幕截图 2024-12-19 133529](https://github.com/user-attachments/assets/b2cd0598-fee1-48a6-8b90-cf4675281cd1)
![屏幕截图 2024-12-19 133723](https://github.com/user-attachments/assets/47a5674f-416e-44c7-ad02-243aced67292)

**Comment by xPaw** [comment]
Use dev build.
